lib/MusicBrainz/Server/Test.pm

```diff
@@ -404,8 +405,7 @@ sub _build_ws_test_json {
             $mech->get_ok($end_point . $url, 'fetching');
             is_valid_json($mech->content, "validating (is_valid_json)");
 
-            is_json($mech->content, $expected);
```

Comment:
I assume it was though of as “the JSON variant of `is`”, not the English statement “is JSON”. Well, it’s gone now.